src/change_rst.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Tlf - contest logging program for amateur radio operators
+ * Copyright (C) 2020 Thomas Beierlein <dl1jbe@darc.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef CHANGE_RST_H
+#define CHANGE_RST_H
+
+void rst_init(char *init) ;
+void rst_reset(void);
+void rst_r_up();
```

Comment:
I'd rename it to `rst_recv_up` as at first glance I thought it's about the R part of the report. Same for `_s_`.

Reply:
I agree with Zoli.  No reason not to have good descriptive variable names.

Reply:
Makes sense. Done.